city_scrapers/spiders/cook_emergency_telephone.py

```diff
@@ -0,0 +1,135 @@
+import re
+from datetime import datetime, timedelta
+from io import BytesIO
+
+import scrapy
+from city_scrapers_core.constants import FORUM
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from PyPDF2 import PdfFileReader
+
+
+class CookEmergencyTelephoneSpider(CityScrapersSpider):
+    name = "cook_emergency_telephone"
+    agency = "Cook County Emergency Telephone System Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.cookcounty911.com/"]
+    location = {
+        "name": "Conference Room",
+        "address": "1401 S. Maybrook Drive, Maywood, IL 60153",
+    }
+
+    def __init__(self, *args, **kwargs):
+        self.meeting_starts = []
+        self.docs_link = ""
+        self.agenda_link = ""
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        schedule_link = ""
+        for link in response.css(".sliding_box a"):
+            link_text = link.css("*::text").get()
+
+            schedule_link = self.start_urls[0] + '/wp-content/uploads/pdfs/'
+
+            if "Minutes" in link_text:
+                self.docs_link = link.attrib["href"]
```

Comment:
It looks like the current "/minutes" page is a list of meeting minutes. We should only attach it to a meeting if we can match the meeting date with a specific minutes link

Reply:
I was thinking of writing a helper function that checks for the meeting date and the minutes date and only then adds it? 
In chi_human_relations they don't check the dates but directly add all minutes links, that is why I used this approach. Which way should I go?

Reply:
That helper function sounds like the way to go. I'd have to double check, but I think the reason we did that in the other spider is because the minutes links were already associated with the date, and it wasn't one page that linked to all of them

Reply:
I might be misunderstanding the changes, but is `docs_link` still needed here?

Reply:
The `docs_link` refers to this page https://cookcounty911.com/minutes/. This page has links to the minutes. We then use this page to parse the links given on the page, if the date for these links match to our meeting we add to the `links_map` as `minutes_href` otherwise these are not added.

Reply:
Got it, thanks for clarifying! That works